tests/ui_tests/reporting/report_master_diff.py

```diff
@@ -0,0 +1,191 @@
+import html
```

Comment:
`html` is a stdlib module unfortunately. perhaps move it together with `import download` and add `# isort:skip`?

Reply:
Hm, or maybe call it `report.py`? But I dont like either

Reply:
both `html` and `download` names are too generic when imported as top-level though? but good names when imported from package.

i'd either move the two imports together, so that it's clearer that they are local
or move the script up a directory and `from reporting import html` to make it extra clear

Reply:
> both html and download names are too generic when imported as top-level though? but good names when imported from package. i'd either move the two imports together, so that it's clearer that they are local

Hm I am not sure these should be merged or that we find a good name for it. But I guess having a generic `report.py` might be fine?

> or move the script up a directory and `from reporting import html` to make it extra clear

Now I am a bit confused. `html.py` is already in `reporting`. Moving the script up a directory means `ui_tests`, which is too generic.